### DISCUSSION TITLE
Кавалорн в круге с ищущими

### DISCUSSION THREAD

**Discussion by dosinabox** [dosinabox/g2nr_unofficial_update#371] [General]
Для мага в третьей главе возможна вот такая забавная ситуация:

![image](https://user-images.githubusercontent.com/39790055/129404011-31314aad-8b20-4c65-a9b4-1dd7c0fc9c08.png)

**Comment by Kor-Angar** [comment]
такая штука есть. можно в таком случае в 3 главе делать проверку и ставить Кавалорна рядом с Лобартом.

**Reply by dosinabox** [reply]
готово
